linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_etc_bashrc/bash/shared.sh

```diff
@@ -8,8 +8,8 @@
 {{% set etc_bash_rc = "/etc/bashrc" %}}
 {{% endif %}}
 
-grep -q umask {{{ etc_bash_rc }}} && \
-  sed -i "s/umask.*/umask $var_accounts_user_umask/g" {{{ etc_bash_rc }}}
+grep -q "^\s*umask" /etc/bashrc && \
```

Comment:
Why was the variable removed?

Reply:
apparently this was an issue with a rebase operation